timesketch/lib/analyzers/gcp_servicekey.py

```diff
@@ -0,0 +1,71 @@
+"""Sketch analyzer plugin for GCP Service Key usage."""
+from __future__ import unicode_literals
+
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+
+
+class GcpServiceKeySketchPlugin(interface.BaseSketchAnalyzer):
+    """Sketch analyzer for GCP Service Key usage."""
+
+    NAME = 'gcp_servicekey'
+
+    def __init__(self, index_name, sketch_id):
+        """Initialize The Sketch Analyzer.
+
+        Args:
+            index_name: Elasticsearch index name
+            sketch_id: Sketch ID
+        """
+        self.index_name = index_name
+        super(GcpServiceKeySketchPlugin, self).__init__(index_name, sketch_id)
+
+    def run(self):
+        """Entry point for the analyzer.
+
+        Returns:
+            String with summary of the analyzer result
+        """
+        query = ('principalEmail:*gserviceaccount*')
+        return_fields = ['message', 'methodName']
+
+        events = self.event_stream(
+            query_string=query, return_fields=return_fields)
+
+        gcp_servicekey_counter = 0
+
+        for event in events:
+            # Fields to analyze.
+            methodName = event.source.get('methodName')
+
+            if 'CreateServiceAccount' in methodName:
+                event.add_star()
```

Comment:
Do we want to star by default? Maybe just label? Also, use a Tag instead of a label. The label is sketch specific and will not follow if the timeline is added to any future sketch.

Reply:
done:
- removed all stars by default
- changed all labels into tags

Reply:
The stars and labels are still there. Did you push? :)